DigitalLearningSolutions.Data/Models/TrackingSystem/EvaluationResponseBreakdown.cs

```diff
@@ -8,14 +8,17 @@ public class EvaluationResponseBreakdown
         public EvaluationResponseBreakdown(string question, IEnumerable<(string response, int count)> responseCounts)
         {
             Question = question;
-            var totalResponses = responseCounts.Sum(x => x.count);
-            ResponseCounts = totalResponses != 0 ? responseCounts : null;
+            var responses = responseCounts.ToList();
+            var totalResponses = responses.Sum(x => x.count);
+            Responses = totalResponses != 0
+                ? responses.Select(x => new EvaluationResponses(x.response, x.count, totalResponses))
+                : null;
         }
 
         public string Question { get; set; }
 
-        public IEnumerable<(string response, int count)>? ResponseCounts { get; set; }
+        public IEnumerable<EvaluationResponses>? Responses { get; set; }

```

Comment:
Don't think you made this, but is there any particular reason this is nullable? Feels like we should just have this as empty by default.

Reply:
Agreed, I see no reason for it being done that way originally. Changed them all to empty lists.